builder/builder.go

```diff
@@ -66,3 +68,26 @@ func (b *Builder) ValidateAgainstPendingState(userTxn *mempool.BroadcastedTransa
 		false, false, false, pendingBlock.Block.GasPrice, pendingBlock.Block.GasPriceSTRK, false)
 	return err
 }
+
+func (b *Builder) GenesisState(genesisConfig GenesisConfig) error {
+	blockTimestamp := uint64(time.Now().Unix())
+	state, closer, err := b.bc.StateAtBlockNumber(0) // Replace with the actual genesis state, that contains the classes etc
```

Comment:
You can use `MakeStateDiffForEmptyBlock` and `blockchain.PendigState` to build a mutable empty state.